### PR TITLE
feat: add grabber for local VBM AOMIC 1000 dataset

### PR DESCRIPTION
**Pull request by FeHoff**
Add a new datagrabber for example dataset from OpenNeuro AOMIC 1000  
VBM is stored on Juseless

### CODE REVIEW

File: junifer/configs/juseless/juseless.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            types=types,
+            datadir=datadir,
+            uri=uri,
+            rootdir=rootdir,
+            replacements=replacements,
+            patterns=patterns,
+        )
+
+@register_datagrabber
+class JuselessDataladAOMICVBM(PatternDataladDataGrabber):
```
Comment (fraimondo) [inline junifer/configs/juseless/juseless.py:47]: Check the linter:

```
  junifer/configs/juseless/juseless.py:46:1: E302 expected 2 blank lines, found 1
  junifer/configs/juseless/juseless.py:63:80: E501 line too long (83 > 79 characters)
```

**Review by fraimondo — CHANGES_REQUESTED** [review]

**Comment by verakye** [comment]
@LeSasse The AOMIC1000 as indicated in issue #60 and the respective pull request in #76 was added to junifer. In the source code you can find the url to the respective dataset where you can see all the data included (https://github.com/juaml/junifer/blob/5ee2f05ccb450584e2d5b297b1f54c28a64eb390/junifer/datagrabber/aomic1000.py#L82). This doesn't include the PIOP1 and PIOP2 datasets.

**Comment by LeSasse** [comment]
> @LeSasse The AOMIC1000 as indicated in issue #60 and the respective pull request in #76 was added to junifer. In the source code you can find the url to the respective dataset where you can see all the data included (
> 
> https://github.com/juaml/junifer/blob/5ee2f05ccb450584e2d5b297b1f54c28a64eb390/junifer/datagrabber/aomic1000.py#L82
> ). This doesn't include the PIOP1 and PIOP2 datasets.

Nice thank you! I will make an issue and implement PIOP1 and 2 based on your ID1000 datagrabber.

File: junifer/configs/juseless/juseless.py (hunk 1 above)
Comment (synchon) [reply junifer/configs/juseless/juseless.py:48]: Should be fixed now.

**Comment by synchon** [comment]
If @FeHoff is okay with the changes, we can merge it @fraimondo after your review.

**Comment by FeHoff** [comment]
> If @FeHoff is okay with the changes, we can merge it @fraimondo after your review.

Looks good to me

**Comment by synchon** [comment]
@fraimondo The changes were valid before #100 and they were mostly `flake8` fixes, so it needs to be adjusted now. Which comment are you referring to?

**Comment by fraimondo** [comment]
Screenshot 2022-10-18 at 11 27 30

**Comment by synchon** [comment]
I see now. It's still Pending, which I'm guessing is probably due to the review not being finished?

I'll take a look and if @FeHoff has already tested then that helps as well.

**Comment by fraimondo** [comment]
Still needs to be refactored to match the current submodule structure.

**Comment by synchon** [comment]
Yes will do that.

**Comment by synchon** [comment]
@fraimondo The refactoring is done and tests are okay, but the juseless test needs to wait until @FeHoff helps us with a solution for the data source as he has recently applied for a change of group for the dataset.

**Comment by FeHoff** [comment]
> @fraimondo The refactoring is done and tests are okay, but the juseless test needs to wait until @FeHoff helps us with a solution for the data source as he has recently applied for a change of group for the dataset.

I only need 26GB of space, where we all have access and I can put the RIA there. I rsync it to `/data/project/deleted_every_sunday/dataladstore/` for testing now

**Comment by synchon** [comment]
> > @fraimondo The refactoring is done and tests are okay, but the juseless test needs to wait until @FeHoff helps us with a solution for the data source as he has recently applied for a change of group for the dataset.
> 
> I only need 26GB of space, where we all have access and I can put the RIA there. I rsync it to `/data/project/deleted_every_sunday/dataladstore/` for testing now

Thanks, I'll test it out using that.

**Comment by fraimondo** [comment]
@FeHoff according to [this](https://juaml.github.io/junifer/main/builtin.html#data-grabbers), the AOMIC ID1000 dataset is open without registration. For this datagrabber to be added, it should be on a location accessible to every person with a juseless account.

**Comment by FeHoff** [comment]
> @FeHoff according to [this](https://juaml.github.io/junifer/main/builtin.html#data-grabbers), the AOMIC ID1000 dataset is open without registration. For this datagrabber to be added, it should be on a location accessible to every person with a juseless account.

I pushed the AOMIC ID1000 CAT data to GIN here: git@gin.g-node.org:/felixh/ds003097_ReproVBM.git
@synchon is testing it now

**Review by fraimondo — APPROVED** [review]
LGTM

**Comment by synchon** [comment]
@FeHoff Thanks for your help! :D

**Pull request merged**